.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -48,3 +48,13 @@ Run the following commands in Powershell and copy/paste the output.
 
 **Additional context**
 <!-- Add any other context about the problem here. -->
+
+**Requested Assignment**
+<!--
+Some people just want to report a bug and let someone else fix.
+Other people want to not only submit the bug report, but want to fix it as well.
+Both scenarios are completely ok.  We just want to know which one it is.
+Please indicate which bucket you fall into by keeping one and removing the other.
+-->
+If possible, I would like to fix this.
+I'm just reporting this problem.  I don't want to fix it.
```

Comment:
Remove extra spaces throughout.

Reply:
Not sure what extra spaces you're referring to.

Reply:
Single space between sentences.